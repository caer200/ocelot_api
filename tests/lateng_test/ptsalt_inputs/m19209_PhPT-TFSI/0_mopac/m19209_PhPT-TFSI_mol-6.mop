CHARGES

 
C	5.757044 +1 1.164845 +1 11.767436 +1
S	5.994356 +1 2.807997 +1 10.978056 +1
F	4.887040 +1 1.213091 +1 12.755174 +1
F	6.890829 +1 0.698126 +1 12.257838 +1
F	5.307646 +1 0.313665 +1 10.857673 +1
N	6.683804 +1 3.664593 +1 12.109679 +1
O	6.970901 +1 2.559458 +1 9.956485 +1
O	4.683417 +1 3.216266 +1 10.584776 +1
S	5.875202 +1 4.440568 +1 13.219541 +1
C	5.701782 +1 6.128692 +1 12.527533 +1
O	4.531156 +1 4.015420 +1 13.447265 +1
O	6.725790 +1 4.618312 +1 14.357692 +1
F	5.013662 +1 6.119245 +1 11.395612 +1
F	5.055897 +1 6.893960 +1 13.400506 +1
F	6.887261 +1 6.669986 +1 12.287444 +1
