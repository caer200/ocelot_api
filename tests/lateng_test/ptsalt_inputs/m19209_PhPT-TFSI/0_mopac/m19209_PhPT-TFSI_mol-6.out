 *******************************************************************************
 ** Site#: 29200        For non-commercial use only    Version 19.090L 64BITS **
 *******************************************************************************
 ** Cite this program as: MOPAC2016, Version: 19.090L, James J. P. Stewart,   **
 ** Stewart Computational Chemistry, web: HTTP://OpenMOPAC.net. Days left: 174**
 *******************************************************************************
 **                                                                           **
 **                                MOPAC2016                                  **
 **                                                                           **
 *******************************************************************************

                              PM7 CALCULATION RESULTS

 *******************************************************************************
 *  CALCULATION DONE:                                Tue Oct  8 20:07:28 2019  *
 *  CHARGES    - IDENTIFY AND PRINT CHARGED ATOMS
 *  T=         - A TIME OF 172800.0 SECONDS REQUESTED
 *  DUMP=N     - RESTART FILE WRITTEN EVERY  7200.000 SECONDS
 *******************************************************************************
CHARGES


   ATOM   CHEMICAL          X               Y               Z
  NUMBER   SYMBOL      (ANGSTROMS)     (ANGSTROMS)     (ANGSTROMS)
 
     1       C          5.75704400  *   1.16484500  *  11.76743600  *
     2       S          5.99435600  *   2.80799700  *  10.97805600  *
     3       F          4.88704000  *   1.21309100  *  12.75517400  *
     4       F          6.89082900  *   0.69812600  *  12.25783800  *
     5       F          5.30764600  *   0.31366500  *  10.85767300  *
     6       N          6.68380400  *   3.66459300  *  12.10967900  *
     7       O          6.97090100  *   2.55945800  *   9.95648500  *
     8       O          4.68341700  *   3.21626600  *  10.58477600  *
     9       S          5.87520200  *   4.44056800  *  13.21954100  *
    10       C          5.70178200  *   6.12869200  *  12.52753300  *
    11       O          4.53115600  *   4.01542000  *  13.44726500  *
    12       O          6.72579000  *   4.61831200  *  14.35769200  *
    13       F          5.01366200  *   6.11924500  *  11.39561200  *
    14       F          5.05589700  *   6.89396000  *  13.40050600  *
    15       F          6.88726100  *   6.66998600  *  12.28744400  *
 


          CARTESIAN COORDINATES 

    NO.       ATOM           X           Y           Z

     1         C          5.7570      1.1648     11.7674
     2         S          5.9944      2.8080     10.9781
     3         F          4.8870      1.2131     12.7552
     4         F          6.8908      0.6981     12.2578
     5         F          5.3076      0.3137     10.8577
     6         N          6.6838      3.6646     12.1097
     7         O          6.9709      2.5595      9.9565
     8         O          4.6834      3.2163     10.5848
     9         S          5.8752      4.4406     13.2195
    10         C          5.7018      6.1287     12.5275
    11         O          4.5312      4.0154     13.4473
    12         O          6.7258      4.6183     14.3577
    13         F          5.0137      6.1192     11.3956
    14         F          5.0559      6.8940     13.4005
    15         F          6.8873      6.6700     12.2874

 General Reference for PM7:
 "Optimization of Parameters for Semiempirical Methods VI: More Modifications to the 
 NDDO Approximations and Re-optimization of Parameters", J. J. P. Stewart, J. Mol. Mod., 1:32, 19 (2013)
 http://www.springerlink.com/openurl.asp?genre=article&id=doi:10.1007/s00894-012-1667-x

           Empirical Formula: C2 N O4 S2 F6  =    15 atoms



      MOLECULAR POINT GROUP   :   C2  

   TOPOGRAPHY OF SYSTEM

   ATOM No.    LABEL   Atoms connected to this atom
      1          C      2      3      4      5
      2          S      7      8      6      1
      3          F      1
      4          F      1
      5          F      1
      6          N      2      9
      7          O      2
      8          O      2
      9          S     11     12      6     10
     10          C      9     13     14     15
     11          O      9
     12          O      9
     13          F     10
     14          F     10
     15          F     10

     Ion Atom No.  Type    Charge
 
    1       6      N       +1

          COMPUTED CHARGE ON SYSTEM: +1

          COMPUTED CHARGE ON SYSTEM: +1

          JOB STOPPED HERE BECAUSE KEYWORD "CHARGES" WAS USED

 ******************************************************************************
 *                                                                            *
 *     Error and normal termination messages reported in this calculation     *
 *                                                                            *
 * JOB STOPPED HERE BECAUSE KEYWORD "CHARGES" WAS USED                        *
 * JOB ENDED NORMALLY                                                         *
 *                                                                            *
 ******************************************************************************



 TOTAL JOB TIME:             0.00 SECONDS

 == MOPAC DONE ==
